coordinator/provider/coordinator.go

```diff
@@ -32,6 +35,139 @@ import (
 	spqrparser "github.com/pg-sharding/spqr/yacc/console"
 )
 
+type FakeClient struct {
```

Comment:
can we just move FakeClient from router/client to pkg/client and reuse it?

Reply:
Nice thought. Done. I also rename it to MockClient